src/sonic-bgpcfgd/bgpcfgd/managers_rm.py

```diff
@@ -72,7 +73,15 @@ def __del_handler_validate(self, key):
         return True
 
     def __update_rm(self, rm, data):
-        cmds = ["route-map %s permit 100" % ("%s_RM" % rm), " set community %s" % data["community_id"]]
-        log_debug("BGPRouteMapMgr:: update route-map %s community %s" % ("%s_RM" % rm, data["community_id"]))
+        cmds = []
```

Comment:
could we add a check if it is FROM_SDN_SLB_ROUTES, to implement these attributes? by this way, we can make it more generic.

Reply:
There is already a check in set_handler() -> __set_handler_validate(), to make sure the key is "FROM_SDN_SLB_ROUTES". Is it possible in future that __set_handler_validate() will allow more route map name string ?

Reply:
@jcaiMR yes, we could extend routemap for others too.

Reply:
ok，will apply in next diff.